package.json

```diff
@@ -89,7 +89,7 @@
     "./util.js": "./util-browser.js"
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=10.0.0"
```

Comment:
If we're dropping support for node 10, this needs to be changed too.

Reply:
Done, thanks.